src/main/java/net/sf/jabref/logic/layout/format/LatexToUnicodeFormatter.java

```diff
@@ -31,6 +31,7 @@
 public class LatexToUnicodeFormatter implements LayoutFormatter, Formatter {
 
     private static final Map<String, String> CHARS = HTMLUnicodeConversionMaps.LATEX_UNICODE_CONVERSION_MAP;
+    private static final Map<String, String> ACCENTS = HTMLUnicodeConversionMaps.UNICODE_ESCAPED_ACCENTS;
```

Comment:
It doesn't matter, as this is a kind of constant which gets replaced by the containing value on compile time, directly in the byte code.
In general, java is  by value,  it copies just the object reference (pointer).
**http://stackoverflow.com/questions/40480/is-java-pass-by-reference-or-pass-by-value**